zanata-war/src/main/java/org/zanata/seam/security/ZanataJpaIdentityStore.java

```diff
@@ -106,13 +106,12 @@ public boolean apiKeyAuthenticate(String username, String apiKey) {
 
     /**
      * Custom authentication that ignores the account's enabled state.
-     *
      * @param username
      * @param password
      * @return
      * @see {@link ZanataJpaIdentityStore#authenticate(String, String)}
      */
-    public boolean authenticateEvenIfDisabled(String username, String password) {
+    public boolean checkPasswordIgnoringActivation(String username, String password) {
```

Comment:
It wasn't really authenticating the user (in terms of changing the state of the user session), so the name was misleading.